resources/views/searches/tasks/buttons/edit_task.blade.php

```diff
@@ -0,0 +1,244 @@
+                                </label>
+                                <input type="text" name="TrackingDevice1" class="form-control">
+                                @error('trackingDevice1')
+                                    <p class="alert alert-danger">{{ $message }}</p>
+                                @enderror
+                            </div>
+
+                            <div class="form-group">
+                                <label for="exampleFormControlFile1">
+                                {{ __('actions.attach') }}</label>
+                                <input type="file" name="GpxFile" class="form-control-file"
+                                accept=".gpx">
+                            </div>
+                        </div>
+                        <div class="col text-left">
+                            <label>-</label>
+                            <input type="text" name="TrackingDevice2" class="form-control">
+                            @error('trackingDevice2')
+                                <p class="alert alert-danger">{{ $message }}</p>
+                            @enderror
+                        </div>
+                    </div>
+
+                        <div class="row mt-3 justify-content-center">
+                            <div class="form-group col-md-12 text-left">
+                                <label class="align-baseline" for="inputDescription">
+                                    {{ __('forms.description') }}
+                                </label>
+                                <textarea name="Description" class="form-control">{{ $task->Description }}</textarea>
+                                @error('Description')
+                                    <p class="alert alert-danger">{{ $message }}</p>
+                                @enderror
+                            </div>
+                        </div>
+                </div>
+                <!-- Modal body - CLOSE -->
+
+                <!-- Modal footer - OPEN -->
+                <div class="modal-footer">
+                    <a class="btn btn-light" data-dismiss="modal">
+                        {{ __('actions.cancel') }}
+                    </a>
+                    <!-- Register button - OPEN -->
+                    <button type="submit" class="btn btn-primary">{{ __('actions.save') }}</button>
+                    <!-- Register button - CLOSE -->
+                </div>
+                <!-- Modal footer - CLOSE -->
+
+            </form>        
+            <!-- Form - CLOSE -->
+
+        </div>
+        <!-- Modal content - CLOSE -->
+
+    </div>
+</div>
+<!-- Edit user modal - CLOSE -->
+
+<!-- JS -->
+<link href="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.28.0/moment.min.js">
+<script type="text/javascript">
+
+    $(document).ready(function() {
+        // today date
+        var today = new Date();
+
+        // begin date input
+        $('.datepicker').daterangepicker({
```

Comment:
Here there should be the validation for the "end" input, not allowing select a date previous to "start" date